content/docs/command-reference/run.md

```diff
@@ -230,15 +230,14 @@ $ dvc run -n my_stage './my_script.sh $MYENVVAR'
   It's used by `dvc repro` to change the working directory before executing the
   `command`.
 
-- `--no-exec` - write the stage to `dvc.yaml`, but do not execute its `command`.
-  Any dependencies and outputs will be entered in `.gitignore`, but won't be
-  cached (like with `--no-commit` below) or recorded in `dvc.lock`. You can use
-  `dvc commit` to save any existing dep/out files to the cache and record their
-  hashes to the lock file.
-
-  This is useful, for example, if you need to define a pipeline quickly (and
-  perhaps share it with others) before executing anything, and run all its
-  stages at once later (with `dvc repro`).
+- `--no-exec` - write the stage to `dvc.yaml`, but do not execute the `command`.
+  DVC will still add the outputs to `.gitignore`, but they won't be cached or
+  recorded in `dvc.lock` (like with `--no-commit` below). You can use
+  `dvc repro` to finish the operation. This is useful if you need to define a
```

Comment:
I think we can omit now ` You can use  dvc repro to finish the operation.`

I wonder if we also simplify the long sentence further somehow ...

Reply:
> we can omit now You can use dvc repro

True, it was repeated. Removed. And I did find a way to simplify the text (dce4cc8b) so it's 20-30% shorter now.

TBH I would completely remove the mentions of --no-cache and `dvc commit` here too. It may be unnecessary (implied/obvious).